### ISSUE TITLE
[bug] copy/paste/undo

### ISSUE THREAD

**Issue by sophiajt** [tauri-apps/tauri#3880]
### Describe the bug

This may be related to https://github.com/tauri-apps/tauri/issues/544, I'm not sure.

I was noticing if I made an `<input>` in a tauri app and typed in it, I wasn't able to do copy/paste or undo inside of that box. If I connect to the same app using the browser it works just fine.

I'm I missing a step to connect these or are they not supported yet?  Thanks

### Reproduction

1. Add `<input>` to your web page for the tauri app
2. Type something
3. Try to copy/paste or undo what you typed

### Expected behavior

All the common edit actions should be supported

### Platform and versions

```shell
/Users/jt/Source/tauri_test/tauri-test〉npm run tauri info                                                                        04/10/2022 04:51:12 PM

> tauri-test@1.0.0 tauri
> tauri "info"


Environment
  › OS: Mac OS 12.3.1 X64
  › Node.js: 17.8.0
  › npm: 8.5.5
  › pnpm: Not installed!
  › yarn: Not installed!
  › rustup: 1.24.3
  › rustc: 1.60.0
  › cargo: 1.60.0
  › Rust toolchain: stable-aarch64-apple-darwin

Packages
  › @tauri-apps/cli [NPM]: 1.0.0-rc.7(outdated, latest: 1.0.0-rc.8)
  › @tauri-apps/api [NPM]: 1.0.0-rc.3(outdated, latest: 1.0.0-rc.3)
  › tauri [RUST]: 1.0.0-rc.6,
  › tauri-build [RUST]: 1.0.0-rc.5,
  › tao [RUST]: 0.7.0,
  › wry [RUST]: 0.14.0,

App
  › build-type: bundle
  › CSP: unset
  › distDir: ../public
  › devPath: http://localhost:8080/
  › framework: Svelte
  › bundler: Rollup

App directory structure
  ├─ node_modules
  ├─ public
  ├─ src-tauri
  ├─ .vscode
  └─ src
```

**Comment by FabianLars** [comment]
duplicate of #2397

tldr: on macOS you need to add a menu with corresponding (native) `MenuItem`s.

**Comment by sophiajt** [comment]
Thanks for the pointer.

It'd be great if this was in the samples and the docs, as the only way I found out about it was your tip. If you search for "copy" or "paste" on the website, you don't turn up this fix.

**Comment by FabianLars** [comment]
We're currently rewriting all of the docs for the incoming v1 stable release, this will be part of it 👍

**Comment by OJFord** [comment]
@FabianLars I'm using 1.0.0, primarily on Linux not macOS, and find that the XF86Paste event doesn't paste, but right-click > paste does. Am I missing something, or should I open an issue about that?

**Comment by FabianLars** [comment]
@OJFord On Linux? Can you create a separate issue for it at the [tao repo](https://github.com/tauri-apps/tao) please?

**Comment by OJFord** [comment]
Yes; thanks. (Future searchers: tauri-apps/tao#475)
